### DISCUSSION TITLE
[DONE ✅]  [GreenField API] Add a Delete User endpoint

### DISCUSSION THREAD

**Discussion by gkrizek** [btcpayserver/btcpayserver#2304] [Ideas / Feature Requests]
Right now the only two user endpoints allow you to create a user and get a user. I'd love to have an API to be able to delete users as well. I'd imagine it would be:

`DELETE /api/v1/users/{userId}`

**Comment by bolatovumar** [comment]
Sounds like a reasonable request. Any reason why this should not be a thing anyone can think of? cc @Kukks

**Comment by Kukks** [comment]
Ack
It hasn't been done so far because you  also can't do it from the current
UI. Should be available only for server adminsp

**Reply by bolatovumar** [reply]
Yeah, I assumed it would only be available to server admins. I guess it could make sense to also add this to the UI?

**Reply by bolatovumar** [reply]
I could take a look into implementing this if no one starts before me (feel free to start work on this before me if I didn't say that I started working on this).

**Reply by gkrizek** [reply]
This does exist in the UI for Admins... I think Andrew was referring to it not existing in the UI for non-admins maybe?

I won't have time to for a while to tackle this so be my guest 😄

**Reply by bolatovumar** [reply]
@gkrizek started a PR here: https://github.com/btcpayserver/btcpayserver/pull/2340

**Comment by gkrizek** [comment]
Yeah I was mainly wanting this as a Server admin. Would be very helpful to have this API! Thanks.
